### DISCUSSION TITLE
local js file import as script tag not working on nuxt 3

### DISCUSSION THREAD

**Discussion by cengizilhan** [nuxt/nuxt#16488] [Questions]
hi guys,

i watching youtube guide videos but its not works for me.
I'm trying add local JS file add to nuxt.

**nuxt.config.ts**
`export default defineNuxtConfig({

  
  head: {
    title: 'My awesome project', 
   script:[
    {
      src: '/js/test.js',
      type: 'text/javascript', // local file add not working..
    }
   ]
   
  },
    modules: [
      '@nuxt/content'
    ],
    content: {
      // https://content.nuxtjs.org/api/configuration
    },
    pageTransition: {
      name: 'fade',
      mode: 'out-in' // default
    },
    layoutTransition: {
      name: 'slide',
      mode: 'out-in' // default
    } ,
    css: ['~/assets/scripts/bootstrap/css/bootstrap.min.css'],

   

  })
  `

script tag not work on nuxt.config.ts .
and i try  app.vue its works on cdn link but not work for local files.
**app.vue**
`<template>
  <NuxtLayout>
    <NuxtPage />
  </NuxtLayout>
</template>

<script>
// /pages/payment-page.vue
export default {
  data () {
    return {
      isStripeLoaded: false
    }
  },
  head () {
    return {
      title: 'Payment Page - My awesome project',
      script: [
        {
          type: 'text/javascript', 
            src: 'https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js',
       },
    // local file add not working..
      ]
    }
  }
}
</script>`

**Comment by Benjack868-coder** [comment]
did you solve this problem sir i have same issue. TIA

**Reply by cengizilhan** [reply]
no. i just install Nuxt 2.
